Common/DTO/UserEmailPassword.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DTO
+{
+    class UserEmailPassword
+    {
+        public string Email { get; }
```

Comment:
How we will create a parametrized instance of it since `new UserEmailPassword() { Email = "", Password = "" }` is blocked?

Reply:
Add public constructor

Reply:
I don`t think about this. Already changed.